modules/generators.py

```diff
@@ -97,7 +97,7 @@ async def _claude_legacy_factory(
 claude_legacy_compat_models = [
     ("claude-3-opus-20240229", "c3o"),
     ("claude-3-sonnet-20240229", "c3s"),
-    ("claude-3-haiku-20240229", "c3h"),
+    ("claude-3-haiku-20240307", "c3h"),
     ("claude-3-5-sonnet-20240620", "c3.5s"),
```

Comment:
is this the old one? can we add the new one too?

Reply:
Added, thanks!